xbmc/Application.cpp

```diff
@@ -3381,12 +3381,13 @@ bool CApplication::IsFullScreen()
 void CApplication::StopPlaying()
 {
   CGUIComponent *gui = CServiceBroker::GetGUI();
-
+  RESOLUTION_INFO res = CServiceBroker::GetWinSystem()->GetGfxContext().GetResInfo();
```

Comment:
why call this here when it's used in the if block? You can also double up the winsystem call
```
auto winSystem = CServiceBroker::GetWinSystem();
```

Reply:
Yes, I experimented and this helps. I will post with changes.